src/contract-access-controller.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+const AccessController = require('./access-controller-interface')
+
+const type = 'eth-contract/my-contract-ac'
+
+class ContractAccessController extends AccessController {
+  constructor(web3, abi, address) {
+    super()
+    this.web3 = web3
+    this.abi = abi
+    // this.contract = new this.web3.eth.Contract(abi, address)
```

Comment:
Disabled for now until we get the web3 part automated or stubbed.

Reply:
:+1: